### PR TITLE
Desktop branch

### PR DESCRIPTION
**Pull request by valt19**
I made some changes in css and html
I updated the desktop-version-branch and made it responsive

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,8 +14,14 @@
...
       </nav>
     </header>
+    <ul class="nav-menu">
+      <li><a href="#portfolio">Portfolio</a></li>
+      <li><a href="#about">About</a></li>
+      <li><a href="#contact">Contact</a></li>
+    </ul>
     <section class="head-section">
     <div class="hey-there">
 <h1 class="hey-text">Hey there. I’m Mariane <br>
```
Comment (mukhammadrizooff) [inline index.html]: - [ ] [REQUIRED] Good job so far! Please give correct style to `Home Page` and also `Navigation` elements like [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=FM3eIBvsAszGSaO5-0)🙏🏼 

image

File: index.html [hunk 2]
```diff
@@ -48,164 +54,206 @@ <h1 class="hey-text">Hey there. I’m Mariane <br>
...
-
-      <div class="recent-works"><h3 class="hey-tex">My Recent Works</h3>
+  <section class="all-works">
+  
+      <div class="recent-works"><h3 class="hey-text">My Recent Works</h3>
       </div>
-        
-
+<div class="green"></div>
+      <div class="project-cards">
```
Comment (mukhammadrizooff) [inline index.html:64]: - [ ] [REQUIRED] Good job so far! Please style the layout of the work cards like [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=FM3eIBvsAszGSaO5-0) 🙏🏼 

image

File: index.html [hunk 3]
```diff
@@ -48,164 +54,206 @@ <h1 class="hey-text">Hey there. I’m Mariane <br>
...
+
+         <textarea id="message" class="msg" name="user_message" placeholder="Enter text here" required maxlength="500"></textarea>
+
+         <button type="submit" class="form-bttn" id="btnid">Get In Touch</button>
+         
+         
+       </form>
+       <footer>
+      </div>
+      <ul class="icons">
```
Comment (mukhammadrizooff) [inline index.html]: - [ ] [REQUIRED] Good job so far! Please give the right style to the `text` on the left and I think you have not created the long `Horizontal` line in the footer, please follow my suggestions above 🙏🏼 

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @valt19, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -34,11 +32,15 @@
   flex: none;
   order: 0;
   flex-grow: 0;
+  margin-left: 15px;
+}
+
+.nav-menu {
+  display: none;
 }
```
Comment (AlphaNtihinduka) [inline style.css:40]: - [ ] As the previous reviewer suggested, kindly try as much as possible to stick to the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=iqWpkNWp3niX7g3e-0). Therefore I suggest that you ensure the color of the nav links match that of Figma design 😎 

Your design 😎 
image

File: style.css [hunk 5]
```diff
@@ -698,11 +709,15 @@ h4 {
   background-color: #017243;
 }
 
-@media screen and (min-width: 768px) {
+@media screen and (min-width: 930px) {
+  .body {
+    width: 100%;
+  }
+
```
Comment (AlphaNtihinduka) [inline style.css]: - [ ] You have done well to ensure that you have the desktop version of the application 👏 . However, as the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#project-requirements) suggest, kindly consider using media queries in your CSS file to make your desktop version look like the template you selected (Desktop: >=768px) ☺️ 
- [ ] Additionally, I have noticed the horizontal scroll in your desktop application. The traditional horizontal scrollbar burdens the user by requiring constant attention and greater physical effort to maintain the dragging 😞 .

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @rosemutai 🙋‍♂️

Good job working on your desktop version of the portifolio. You have done the design to the best expectation and incredibly well. 👏  
 
You still need to implement few on the project but you are almost there!!! 🚶 : 

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @valt19, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ABDULALI3468 — COMMENTED** [review]
Hi @valt19 , 👍 

# APPROVED 💯 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights 
☑️ Approving PR already approved before.
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**